build/configs/artik05x/artik05x_cmn.sh

```diff
@@ -34,10 +34,10 @@ CODESIGNER_PATH=${ARTIK05X_DIR_PATH}/tools/codesigner
 CODESIGNER_DIR_PATH=${ARTIK05X_DIR_PATH}/tools/codesigner
 OPENOCD_DIR_PATH=${BUILD_DIR_PATH}/tools/openocd
 
-if [[ $CONFIG_HOST_OSX == 'y' ]]; then
+if [[ $OSTYPE == "darwin"* ]]; then
```

Comment:
Where is OSTYPE definition?

Reply:
It is bash variable

Reply:
This variable will  auto set to a string that describes the OS on platform where bash execution happens